pkgs/top-level/python-packages.nix

```diff
@@ -7386,9 +7392,7 @@ in {
     cudatoolkit = pkgs.cudatoolkit_11_0;
     cudnn = pkgs.cudnn_cudatoolkit_11_0;
     nccl = pkgs.nccl_cudatoolkit_11;
-    openssl = pkgs.openssl_1_1;
     inherit (pkgs.darwin.apple_sdk.frameworks) Foundation Security;
-    inherit (pkgs) flatbuffers;
```

Comment:
Please revert this.

Reply:
Changed the names to avoid the conflict, see above.